source/Calamari.Common/Features/Packages/PackageName.cs

```diff
@@ -46,6 +46,14 @@ public static string[] ToSearchPatterns(string packageId, IVersion? version = nu
                 .Distinct()
                 .ToArray();
         }
+        
+        /// <summary>
+        /// The PackageId could be supplied as a path within a Feed, e.g a Helm OCI feed allows a feed pointing at the root, but packages can be stored with paths:
+        /// e.g
+        /// - package1.tgz
+        /// - folder/package2.tgz - the package id for this package would be folder/package2
+        /// </summary>
+        public static string ExtractPackageNameFromPathedPackageId(string packageId) => packageId.Contains("/") ? packageId.Split(new[]{'/'}, StringSplitOptions.RemoveEmptyEntries).Last() : packageId;
```

Comment:
This can be simplified, `.Split` will just return the original value if there's no `/`.

```suggestion
        public static string ExtractPackageNameFromPathedPackageId(string packageId) => packageId.Split(new[]{'/'}, StringSplitOptions.RemoveEmptyEntries).Last();
```